doc/Index.md

```diff
@@ -85,4 +85,4 @@ VS Project System Documentation
   - [Add Single File Generator](scenario/add_single_file_generator.md)
 - [Differences between Visual Studio 2013 and Visual Studio 2015](overview/VS2013_vs_VS2015.md)
 - [Changes](../CHANGES.md)
```

Comment:
While were here, we should move this item up and maybe rename it for clarity

Reply:
I grouped all breaking changes together